documentation/source/newsfragments/1478.bugfix.rst

```diff
@@ -0,0 +1 @@
+fix Verilator not writing coverage files in some cases
```

Comment:
```suggestion
Fixed Verilator not writing coverage files in some cases
```
Apologies for nitpicking but these end up in the release notes.

Reply:
my apologies, i missed this on the first request.